docs/src/scss/custom.scss

```diff
@@ -248,11 +248,13 @@ footer.footer {
   background-position: top left;
   background-size: cover;
   border-top: 1px solid $blue-300;
+  padding-bottom: 0px;
   .footer__bottom {
     display: flex;
     align-items: center;
     border-top: 1px solid $blue-300;
     padding: 16px 0 8px;
+    margin-top: 60px;
```

Comment:
Can we try with em/rem ?

Reply:
In this file, we are using `px` unit for all margins and paddings. I think we should keep same unit